<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="2" total="4" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite1" duration-ms="84" started-at="2020-05-05T20:54:27 IST" finished-at="2020-05-05T20:54:27 IST">
    <groups>
    </groups>
    <test name="Test2" duration-ms="84" started-at="2020-05-05T20:54:27 IST" finished-at="2020-05-05T20:54:27 IST">
      <class name="login.Login_Neg01">
        <test-method status="PASS" signature="Setup()[pri:0, instance:login.Login_Neg01@1e4a7dd4]" name="Setup" is-config="true" duration-ms="115" started-at="2020-05-05T20:54:27 IST" finished-at="2020-05-05T20:54:27 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Setup -->
        <test-method status="FAIL" signature="testNegative()[pri:0, instance:login.Login_Neg01@1e4a7dd4]" name="testNegative" duration-ms="20" started-at="2020-05-05T20:54:27 IST" finished-at="2020-05-05T20:54:27 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at login.Login_Neg01.testNegative(Login_Neg01.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNegative -->
        <test-method status="PASS" signature="getResult(org.testng.ITestResult)[pri:0, instance:login.Login_Neg01@1e4a7dd4]" name="getResult" is-config="true" duration-ms="22" started-at="2020-05-05T20:54:27 IST" finished-at="2020-05-05T20:54:27 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testNegative status=FAILURE method=Login_Neg01.testNegative()[pri:0, instance:login.Login_Neg01@1e4a7dd4] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:login.Login_Neg01@1e4a7dd4]" name="teardown" is-config="true" duration-ms="361" started-at="2020-05-05T20:54:27 IST" finished-at="2020-05-05T20:54:28 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- login.Login_Neg01 -->
      <class name="login.Login_Neg02">
        <test-method status="FAIL" signature="testNegative()[pri:0, instance:login.Login_Neg02@4f51b3e0]" name="testNegative" duration-ms="7" started-at="2020-05-05T20:54:27 IST" finished-at="2020-05-05T20:54:27 IST">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at login.Login_Neg02.testNegative(Login_Neg02.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testNegative -->
        <test-method status="FAIL" signature="getResult(org.testng.ITestResult)[pri:0, instance:login.Login_Neg02@4f51b3e0]" name="getResult" is-config="true" duration-ms="11" started-at="2020-05-05T20:54:27 IST" finished-at="2020-05-05T20:54:27 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testNegative status=FAILURE method=Login_Neg02.testNegative()[pri:0, instance:login.Login_Neg02@4f51b3e0] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at login.BaseTest.getResult(BaseTest.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:63)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:348)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:695)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:671)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:655)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
      </class> <!-- login.Login_Neg02 -->
    </test> <!-- Test2 -->
  </suite> <!-- Suite1 -->
</testng-results>
